app/src/main/java/com/conupods/OutdoorMaps/BuildingDataMap.java

```diff
@@ -141,7 +141,7 @@ private void logAllCLassrooms(Building building) {
         while (jsonReader.hasNext()) {
             String classRoom = jsonReader.nextString();
             classRooms.add(classRoom);
-            Log.d(TAG, classRoom);
+            // Log.d(TAG, classRoom);
```

Comment:
is this necessary to keep

Reply:
it wasn't my code, i just commented it out in case they need it so it wouldnt spam my tests

Reply:
We can remove it, im the one who wrote it